cmd/new/new.go

```diff
@@ -68,10 +66,6 @@ func validateArgCount(cmd *cobra.Command, args []string) error {
 		listTemplates = true
 	}
 
-	if targetName == "" && len(args) == 2 {
```

Comment:
I need to add more code comments here. This line was added to account for `pct new puppet-class foo`.  There may be better ways of handling that with cobra, but we shouldn't remove the functionality.

I also need to start a UX sheet of command workflows we support.

Reply:
added this back in but as discussed this is more magic